setup.cfg

```diff
@@ -29,6 +29,7 @@ console_scripts =
    pbench-clear-results = pbench.cli.agent.commands.results.clear:main
    pbench-clear-tools = pbench.cli.agent.commands.tools.clear:main
    pbench-config = pbench.cli.agent.commands.conf:main
+   pbench-reindex = pbench.cli.server.reindex:reindex
    pbench-report-generator = pbench.cli.server.report:report
    pbench-tree-manage = pbench.cli.server.tree_manage:tree_manage
    pbench-is-local = pbench.cli.agent.commands.is_local:main
```

Comment:
Huh? I inserted it properly with respect to "config", "report-generator", and "tree-manage" -- but you're correct that the old "is-local" as some point became out of order, possibly because I originally got `pbench-tree-manage` out of order? 😦 

And all of these arguably should have gone further down the list after `pbench-register-tool-trigger`, among existing entries. I'm not sure how this started, much less how it will end. 😆